extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/core-configuration.xml

```diff
@@ -676,6 +676,23 @@
           <value>org.exoplatform.social</value>
         </value-param>
       </init-params>
+    </component-plugin>  
+  </external-component-plugins>
+  
+  <external-component-plugins>
```

Comment:
In order to enable the upgrade plugin, it will be put in the upgrade.properties file.

Pls have a look here:

https://github.com/exodev/platform/blob/develop/extension/config/src/main/resources/conf/platform/upgrade-4.2.1-to-4.3.0-RC1-sample.properties